src/components/Dashboard/Dashboard.styles.js

```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components';
+
+export const Article = styled.h2`
```

Comment:
Why not rename this styled component to `DarkTitle` or something more meaningful or semantic?

Reply:
I am changing it